typescript/cli/src/deploy/warp.ts

```diff
@@ -80,6 +81,25 @@ export async function runWarpRouteDeploy({
     warpRouteDeploymentConfigPath,
   );
 
+  const context = dryRun
+    ? await getDryRunContext({
+        chainConfigPath,
+        chains: [warpRouteConfig.base.chainName],
+        coreConfig: { coreArtifactsPath },
+        keyConfig: { key },
+        skipConfirmation,
+      })
+    : await getContext({
+        chainConfigPath,
+        coreConfig: { coreArtifactsPath },
+        keyConfig: { key },
+        skipConfirmation,
+      });
+
+  const multiProvider = context.multiProvider;
+  const signer = context.signer;
+  const coreArtifacts = context.coreArtifacts;
```

Comment:
Nit: could be `const {mp, signer, core} = context`

Reply:
Ah yep– I was initially re-assigning to one of these fields but no longer relevant. Reverted.